lib/mongorito.js

```diff
@@ -25,6 +16,14 @@ const is = require('is_js');
 
 const Query = require('./query');
 
+const MongoClient = mongodb.MongoClient;
+
+class EnforcerError extends Error {
```

Comment:
I am definitely against using symbols and this weird stuff to implement a singleton pattern. It would be useful, if you mentioned what kind of errors did you have. This PR changes the underlying driver, not the API, so it should've worked just fine.